hydpy/models/rconc_nash.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""Calculate runoff concentration by linear storage cascade model ('Nash cascade').
+
+|rconc_nash| is a submodel that supplies its main model with the
+calculation of the runoff concentration by the storage cascade approach.
```

Comment:
|rconc_nash| is a submodel that supports its main model by calculating the runoff concentration using the storage cascade approach.

Reply:
ok